LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
There's a thread in flowdock which I can't find as it's not on the process channel where I expected it.

@lekkas There was a discussion about this recently, can you remember the ins and outs?